modules/storages/app/workers/storages/manage_nextcloud_integration_events_job.rb

```diff
@@ -30,22 +30,23 @@ module Storages
   class ManageNextcloudIntegrationEventsJob < ApplicationJob
     include ManageNextcloudIntegrationJobMixin
 
-    DEBOUNCE_TIME = 5.seconds.freeze
+    THREAD_DEBOUNCE_TIME = 4.seconds.freeze
+    JOB_DEBOUNCE_TIME = 5.seconds.freeze
     THREAD_KEY = :manage_nextcloud_integration_events_job_debounce_happend_at
```

Comment:
I guess `THREAD_KEY` is not the correct naming anymore, now that we do not fiddle around with threads. Maybe `StoreKey`?

Reply:
I believe having THREAD terminology is fine. It highlights that debounce time operates in a thread scope. RequestStore basically uses Thread variables as well. The only difference is that it cleans them up at the end of request cycle in the middleware. So, I suggest having the following names: 
```
    THREAD_DEBOUNCE_TIME = 4.seconds.freeze
    JOB_DEBOUNCE_TIME = 5.seconds.freeze
    REQUEST_STORE_KEY = :manage_nextcloud_integration_events_job_debounce_happend_at
```
Will it be good for you?